share/spice/twitter/skipwords.txt

```diff
@@ -4,4 +4,12 @@ users?
 search(es)?
 analytics?
 company
-about
\ No newline at end of file
+about
+signup
+login
+app(s)?
+download
+make
+open
+help
+setting(s)?
```

Comment:
@nalinbhardwaj also please add, "logo", "stock", "emoji", "emoticons"

And please alpha-sort this list 👍